src/main/resources/tag_profile.yaml

```
@@ -30,20 +30,26 @@ tagMetrics:
   - tag: OpenShift-metrics
     metricId: redhat.com:openshift_container_platform:cpu_hour
     uom: CORES
-    prometheusMetric: cluster:usage:workload:capacity_physical_cpu_cores:max:5m
-    prometheusMetadataMetric: subscription_labels
+    queryParams:
+      product: ocp
+      prometheusMetric: cluster:usage:workload:capacity_physical_cpu_cores:max:5m
+      prometheusMetadataMetric: subscription_labels{product="#{tag.queryParams[product]}", ebs_account="#{runtime[account]}", billing_model="marketplace", support=~"Premium|Standard|Self-Support|None"}
```

I'd like to see this restructured a bit:

```suggestion
      prometheusMetadataMetric: subscription_labels
```

and then in the query template:

```
max(sum_over_time(#{tag.queryParams[prometheusMetric]}[1h:5m]) / 13.0) by (_id)
            * on(_id) group_right
            min_over_time(#{tag.queryParams[prometheusMetadataMetric]}{product="#{tag.queryParams[product]}", ebs_account="#{runtime[account]}", billing_model="marketplace", support=~"Premium|Standard|Self-Support|None"}[1h])
```

That way account query template can be:

```
group(min_over_time(#{#{tag.queryParams[prometheusMetadataMetric]}}{product='#{tag.queryParams[product]}', ebs_account != '', billing_model='marketplace'}[1h]
```
.
Also, note that we're planning on make the product label optional, so that will have some impact here, but would rather handle outside of scope of this PR.

---

DONE - This really cleans things up a lot. 

> Also, note that we're planning on make the product label optional, so that will have some impact here, but would rather handle outside of scope of this PR.

ACK. Don't think this will be a big deal when we have to consider this.